api/web3server.go

```diff
@@ -397,7 +397,7 @@ func (svr *Web3Server) getTransactionCount(in interface{}) (interface{}, error)
 	if err != nil {
 		return nil, err
 	}
-	return uint64ToHex(accountMeta.Nonce), nil
+	return uint64ToHex(accountMeta.GetPendingNonce()), nil
```

Comment:
bug introduced in https://github.com/iotexproject/iotex-core/commit/79892232b299c056900788e40fbaf7c0b2457fd7, which is reverting https://github.com/iotexproject/iotex-core/commit/a35086f498d4841191b79154cdab74d1aa222bfe

if `git revert` is used, should have prevented this bug